releasenotes/notes/fix-some-pyflakes-errors-69a1106efa5d0203.yaml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Can you remove this file? I added the label, the CI job will pass.

Reply:
Done.